### DISCUSSION TITLE
login into docker container

### DISCUSSION THREAD

**Discussion by wcizmowski** [dunglas/symfony-docker#347] [General]
Hi,
Whats the way to login into docker container ?

Below link do not help me:
https://phase2.github.io/devtools/common-tasks/ssh-into-a-container/

And I test:
[witek@ /home/witek/Projects/symfony-docker ] docker login localhost:8000
Username: root
Password: 
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/auth": dial unix /var/run/docker.sock: connect: permission denied
[witek@ /home/witek/Projects/symfony-docker ] docker login localhost:8000
Username: admin
Password: 
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/auth": dial unix /var/run/docker.sock: connect: permission denied
[witek@ /home/witek/Projects/symfony-docker ] 

I dont know user and password.

But probably another metod exist...

Please help, I need login into docekr container

**Comment by maxhelias** [comment]
Use the real docker documentation : https://docs.docker.com/

There is a difference between login into a docker registry and a docker container : 
 - https://docs.docker.com/engine/reference/commandline/login/
 - https://docs.docker.com/engine/reference/commandline/exec/

And an other with compose : 
- https://docs.docker.com/engine/reference/commandline/compose_exec/
